src/main/webapp/script.js

```diff
@@ -173,6 +173,42 @@ function getGraphDisplay(graphNodes, graphEdges) {
     const node = evt.target;
     node.tip.show();
   });
+
+  let searchElement = document.getElementById('search');
+  searchElement.onchange = function() {
```

Comment:
You should consider add tests for this! To make sure the innerText is being updated correctly?